src/components/ha-picture-upload.ts

```diff
@@ -250,6 +250,11 @@ export class HaPictureUpload extends LitElement {
           max-height: 200px;
           margin-bottom: 4px;
           border-radius: var(--file-upload-image-border-radius);
+          transition: opacity 0.3s;
+          opacity: var(--picture-opacity, 1);
+        }
+        img:hover {
+          opacity: 1 !important;
```

Comment:
No need for important I think?

Reply:
Yeah it's not needed anymore. I was useful because it was using opacity style. We can remove the important flag.